pkg/utils/globalTypes/food/food.go

```diff
@@ -0,0 +1,3 @@
+package food
+
+type FoodType = int
```

Comment:
I think this is aliasing instead of typedef-ing

```suggestion
type FoodType int
```

Reply:
Agree actually, this is better.

Reply:
yeah I know, does it matter though? Imo changing this to a typedef just means that you would have to cast everything to food.FoodType which means you can't just use the underlying type directly, e.g. `a.TakeFood(rand.Intn(100))` would need to be `a.TakeFood(food.FoodType(rand.Intn(100)))`. I'm perfectly fine with changing it to a typedef but just bear in mind that you'll have to ensure that the types match

Reply:
I still think the suggested change is better

Reply:
okay sure, i'll change it